### PR TITLE
deprecate media upload functions in helpers

### PR DESCRIPTION
**Pull request by mharsat**

### CODE REVIEW

File: packages/editor/web/src/RichContentEditor/RichContentEditor.jsx [hunk 1]
```diff
@@ -269,6 +275,43 @@ class RichContentEditor extends Component {
...
+      handleFileSelection: galleryConfig?.handleFileSelection,
+      ...imageConfig,
+    };
+    config[GALLERY_TYPE] = {
+      handleFileUpload: imageConfig?.handleFileUpload,
+      handleFileSelection: imageConfig?.handleFileSelection,
+      ...galleryConfig,
+    };
+  }
+
```
Comment (shaulgo) [inline packages/editor/web/src/RichContentEditor/RichContentEditor.jsx:291]: remove this

File: packages/editor/web/src/RichContentEditor/RichContentEditor.jsx [hunk 2]
```diff
@@ -269,6 +275,43 @@ class RichContentEditor extends Component {
...
+      config[SOUND_CLOUD_TYPE].onVideoSelected =
+        soundCloudConfig?.onVideoSelected ||
+        videoConfig?.onVideoSelected ||
+        galleryConfig?.onVideoSelected;
+    if (galleryConfig)
+      config[GALLERY_TYPE].onVideoSelected =
+        galleryConfig?.onVideoSelected ||
+        videoConfig?.onVideoSelected ||
+        soundCloudConfig?.onVideoSelected;
+  }
```
Comment (shaulgo) [inline packages/editor/web/src/RichContentEditor/RichContentEditor.jsx:313]: remove

File: examples/storybook/stories/Plugins/Image/ImagePluginStory.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ import SyntaxHighlighter from '../../Components/SyntaxHighlighter';
 import { mockImageNativeUploadFunc } from '../../../../main/shared/utils/fileUploadUtil';
 
 const mockErrorMsg = 'file too large';
-const onFilesChangeMap = {
+const handleFileUploadMap = {
```
[the remark this answers is not in the document]
Comment (mharsat) [reply examples/storybook/stories/Plugins/Image/ImagePluginStory.js:18]: very strange map object

File: packages/common/web/lib/deprecateHelpers.js [hunk 4]
```diff
@@ -1,15 +1,49 @@
...
+    }
+
+    delete helpers.handleFileSelection;
+  }
+
+  if (onVideoSelected) {
+    if (config[VIDEO_TYPE]) {
+      config[VIDEO_TYPE].onVideoSelected = onVideoSelected;
+      config[SOUND_CLOUD_TYPE].onVideoSelected = onVideoSelected;
+      config[GALLERY_TYPE].onVideoSelected = onVideoSelected;
```
[the remark this answers is not in the document]
Comment (mharsat) [reply packages/common/web/lib/deprecateHelpers.js:44]: oooh nice catch

**Pull request merged**